src/containers/EditListingPage/EditListingWizard/EditListingDetailsPanel/EditListingDetailsForm.js

```diff
@@ -115,6 +115,111 @@ const FieldSelectListingType = props => {
   );
 };
 
+// Finds the correct subcategory within the given categories array based on the provided categoryIdToFind.
+const findCorrectSubcategory = (categories, categoryIdToFind) => {
```

Comment:
Rename: `findCategoryConfig`
This finds the category config for given categoryId.